mteb/tasks/Reranking/rus/WebLINXCandidatesRerankingRu.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import annotations
+
+import datasets
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskReranking import AbsTaskReranking
+
+
+class WebLINXCandidatesRerankingRu(AbsTaskReranking):
+    metadata = TaskMetadata(
+        name="WebLINXCandidatesReranking",
+        description="WebLINX is a large-scale benchmark of 100K interactions across 2300 expert demonstrations of conversational web navigation. The reranking task focuses on finding relevant elements at every given step in the trajectory.",
+        reference="https://mcgill-nlp.github.io/weblinx",
+        dataset={
+            "path": "DeepPavlov/WebLINX-ru",
+            "name": "reranking",
+            "revision": "fdc93113fd2d43ee6aae0fd59a53ebd7bc201287",
+        },
+        type="Reranking",
+        category="p2p",
+        modalities=["text"],
+        eval_splits=[
+            "validation",
+            "test_iid",
+            "test_cat",
+            "test_geo",
+            "test_vis",
+            "test_web",
+        ],
+        eval_langs=["rus-Cyrl"],
+        main_score="mrr",
```

Comment:
В `v2` версии надо поменять
```suggestion
        main_score="mrr_at_10",
```